src/validators/ClassValidator.ts

```diff
@@ -234,6 +236,37 @@ export class BsClassValidator {
         }
     }
 
+
+    /**
+     * Check the types for fields, and validate they are valid types
+     */
+    private validateFieldTypes() {
+        for (let key in this.classes) {
+            let classStatement = this.classes[key];
+            classStatement.body.filter(statement => isClassFieldStatement(statement))
+                .forEach((statement) => {
```

Comment:
Could you refactor this to use a `for(let statement of statements)` loop instead of `Array.forEach`? I know this one is a little nit-picky, but `Array.forEach` is _slightly_ less efficient than a raw for loop, and ClassValidator is a very frequently used piece of code so we want to keep it as fast as possible.